app/pods/newsletter/nota-updates/route.js

```diff
@@ -0,0 +1,111 @@
+import Route from '@ember/routing/route';
+import moment from 'moment';
+import { hash } from 'rsvp';
+import { A } from '@ember/array';
+import CONFIG from 'fe-redpencil/utils/config';
+import getPaginationMetadata from 'fe-redpencil/utils/get-pagination-metadata';
+import {
+  task, timeout
+} from 'ember-concurrency';
+
+
+export default class NewsletterNotaUpdatesRoute extends Route {
+  queryParams = {
+    sort: {
+      refreshModel: true,
+    },
+  };
+
+  @(task(function *() {
+    while (true) {
+      yield timeout(0.1 * 60000);
```

Comment:
Ik weet niet of er hieromtrent iets afgesproken is, maar elke 6 seconden lijkt me overdreven ... ik zou gaan voor elke 5 minuten, hoogstens elke minuut ...

Reply:
Ik heb 3 minuten genomen.. we checken af met de gebruikers hoeveel dit moet zijn